### ISSUE TITLE
[Bug]: NOT A BUG BUT A FEATURE: Log server on windows platform.

### ISSUE THREAD

**Issue by Shituation** [OpenTTD/OpenTTD#10649]
### Version of OpenTTD

13.X

### Expected result

DISCLAIMER: the link to discussions (old forum) doesn't let me search for "log" because the word is too short. So here I am.

Need a way to log verbose output from the openttd exe to a file. Tried the suggested methods from the forum https://www.tt-forums.net/viewtopic.php?t=75856 and https://www.tt-forums.net/viewtopic.php?f=65&t=75648,  none of them worked, I only end up with an empty logfile.

### Actual result

I can't have a log and see what happened after a server restart.

### Steps to reproduce

Set up a dedicated server on windows.

**Comment by Shituation** [comment]
Made this because that discussion place is dead as it can be. If you look there, I appear as "the most helpful" because I had to auto-answer myself. I mean, if it had any activity, there would be other people as "the most helpful". 

The old forum is dead and restricts searches of short words.

No dev here seems to care so much about anything (all requests very frequently dismissed)

I waited years to have passwords saved on multiplayer games.
